policyengine_us/parameters/gov/irs/credits/estate/base.yaml

```diff
@@ -0,0 +1,12 @@
+description: The IRS provides this base exclusion amount under the estate tax credit. 
+metadata:
+  unit: currency-USD
+  label: Estate tax credit base exclusion amount  
+  period: year
+  reference:
+    - title: 26 U.S. Code § 2010 - Unified credit against estate tax - (c)(3)(A)
+      href: https://www.law.cornell.edu/uscode/text/26/2010
```

Comment:
are there tax form that we can use? 2021, 2022, 2023

Reply:
https://www.irs.gov/businesses/small-businesses-self-employed/whats-new-estate-and-gift-tax
I found this website which seems has a totally different base amount

Reply:
It should be one of these:
https://www.irs.gov/businesses/small-businesses-self-employed/forms-and-publications-estate-and-gift-tax

Reply:
Looks like the [form 706 instructions](https://www.irs.gov/pub/irs-pdf/i706.pdf)

Reply:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/ab4e5a5f-1865-4e32-87eb-00701336680f)
Rounding rules